astropy/table/table.py

```diff
@@ -588,7 +588,8 @@ def _base_repr_(self, html=False):
         tableid = 'table{id}'.format(id=id(self))
         data_lines, outs = self.formatter._pformat_table(self,
             tableid=tableid, html=html, max_width=(-1 if html else None),
-            show_name=True, show_unit=None, show_dtype=True)
+                                                    show_name=True, show_unit=None,
+                                                         show_dtype=True)
```

Comment:
this is just nitpicking: as the indentation of 'tableid' was brought back to the beginning of the line (rather than under 'self'), this should be aligned with that one, too (and thus you don't need two lines to fit in the 80 char line length)